test/controllers/config_reports_controller_test.rb

```diff
@@ -44,6 +44,33 @@ def test_index
     assert_equal 1, response.body.lines.size
   end
 
+  test 'Ansible origin and icon are identified' do
+    create_a_report_with_origin('Ansible')
+    get :index, params: {format: :json}, session: set_session_user
+    parsed = YAML.safe_load(response.body)
+    assert_response :success
+    assert_equal 'Ansible', parsed['reports'][0]['origin']['label']
+    assert_match /\/assets\/Ansible-[0-9a-f]+\.png/, parsed['reports'][0]['origin']['src']
+  end
+
+  test 'Puppet origin and icon are identified' do
+    create_a_report_with_origin('Puppet')
+    get :index, params: {format: :json}, session: set_session_user
+    parsed = YAML.safe_load(response.body)
+    assert_response :success
+    assert_equal 'Puppet', parsed['reports'][0]['origin']['label']
+    assert_match /\/assets\/Puppet-[0-9a-f]+\.png/, parsed['reports'][0]['origin']['src']
+  end
+
+  test 'Salt origin and icon are identified' do
+    create_a_report_with_origin('Salt')
+    get :index, params: {format: :json}, session: set_session_user
+    parsed = YAML.safe_load(response.body)
+    assert_response :success
+    assert_equal 'Salt', parsed['reports'][0]['origin']['label']
+    assert_match /\/assets\/Salt-[0-9a-f]+\.png/, parsed['reports'][0]['origin']['src']
```

Comment:
You can also use `%r{}` to avoid the need to escape slashes.
```suggestion
    assert_match %r{/assets/Salt-[0-9a-f]+\.png}, parsed['reports'][0]['origin']['src']
```

Reply:
Done, thanks!